.python-version

```diff
@@ -0,0 +1 @@
+2.7.16
```

Comment:
I poked around a bit to find that it looks like we do specify python2.6 [here](https://github.com/mozilla-services/docs/blob/master/source/Makefile#L42) so if there are concerns with the discrepancy I can tweak this to match.

Reply:
Thanks, I expect that build target is left over from something we no longer do and can just be removed. Which is to say, I don't have any concerns with the discrepancy :-)